### PR TITLE
PMM-7583 Google Cloud Platform instructions for MySQL and PostgreSQL

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
WIP

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
Pending decision on whether to include 'Cloud SQL Proxy' section.

**Comment by rnovikovP** [comment]
I don't see a reason to remove the Cloud SQL section if we investigated and described it already.
But I can't review the document itself, I expect Dev team can take care of it

**Comment by PaulJacobs-percona** [comment]
https://docs.google.com/document/d/1bdipdzzyVnQ1jdofTZSjJ0ZiVgZw3p76EOxGHyU2h8Q/edit?usp=sharing
Google docs version of page

**Review by askomorokhov — APPROVED** [review]

**Review by denisok — APPROVED** [review]

File: docs/setting-up/client/google.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+PMM can monitor MySQL or PostgreSQL instances hosted on the [Google Cloud Platform][GOOGLE_CLOUD].
+
+The connection can be direct, or indirect using [Cloud SQL Proxy][GOOGLE_CLOUD_SQL_PROXY].
+
+## MySQL
+
+1. [Set up a MySQL instance on Google Cloud][GOOGLE_CLOUD_MYSQL]
+
+2. The database server must be accessible by PMM Client. If PMM Client is not also hosted on GCP, you will need to add a network interface with a public interface.
```
Comment (denisok) [inline docs/setting-up/client/google.md:11]: do we have documentation about it already, or maybe link to external doc about how to add it ?

**Review by denisok — APPROVED** [review]

File: docs/setting-up/client/google.md (hunk 1 above)
Comment (PaulJacobs-percona) [reply docs/setting-up/client/google.md:11]: We decided not to explain how to do that, as it's specific to GCP and not PMM. With these cloud page (AWS, Azure, GCP) we have to assume the reader knows how to manage their own service.

**Pull request merged**